src/features/channels/components/ChannelsEdit.tsx

```diff
@@ -132,15 +136,16 @@ const BaseChannelsEdit = ({
           borderRadius: "0px",
           padding: "15px 21px",
           display: "flex",
-          justifyContent: "center"
+          justifyContent: "flex-start"
         }}
       >
-        <StyledButtonPrimary
+        <StyledButton
+          color="secondary"
```

Comment:
This line changes this button to use the secondary rather than the primary color palette. It is a not a typo. According to the designs, two of the buttons that were styled as primary (Add Package, Add Channel) should be styled as secondary.